src/OrchardCore/OrchardCore.ContentManagement/DefaultContentManager.cs

```diff
@@ -227,7 +227,8 @@ public async Task<ContentItem> GetAsync(string contentItemId, VersionOptions opt
                     var contentTypeDefinition = _contentDefinitionManager.GetTypeDefinition(contentItem.ContentType);
 
                     // Check if not versionable, meaning we use only one version
-                    if (!(contentTypeDefinition?.GetSettings<ContentTypeSettings>().Versionable ?? true))
+
+                    if (contentTypeDefinition == null || !contentTypeDefinition.IsVersionable())
```

Comment:
Why we change the syntax here, it could be, or is there something wrong

```csharp
contentTypeDefinition?.IsVersionable() ?? true
```

Reply:
`contentTypeDefinition?.IsVersionable() ?? true` is invalid since we are checking for NOT versinable. The logic will have to be `!(contentTypeDefinition?.IsVersionable() ?? true)` which is much harder to read than what I have. Fancier? Maybe, but I always take readability over fancy.

Reply:
Hmm but it seems that before the condition was false if null

    definition == null => !(true) => false

So to repro the exact same behavior it seems that you should use

    if (contentTypeDefinition != null && !contentTypeDefinition.IsVersionable())

Unless it was already wrong before ;)

Reply:
@jtkech good point and catch! I fixed it